themes/default/content/blog/how-we-use-pulumi-to-build-pulumi/index.md

```diff
@@ -175,7 +175,7 @@ We serve three properties as static content from S3, each described
 using Pulumi. One of these is
 [https://get.pulumi.com](https://get.pulumi.com/) which hosts downloads
 of our `pulumi` CLI installer. It's a very simple program, and even
-better - it's open source at <http://github.com/pulumi/get.pulumi.com>.
+better - it's open source at [GitHub](https://github.com/pulumi/get.pulumi.com).
```

Comment:
Suggest a slight tweak to this just for readability (i.e., should read "on GitHub" rather than "at GitHub"), and including the words "open source on" in the link slightly clarifies that we aren't just linking to the GitHub home page.  

```suggestion
better - it's [open source on GitHub](https://github.com/pulumi/get.pulumi.com).
```